### ISSUE TITLE
Wrong journeys due to very inadequate walking time

### ISSUE THREAD

**Issue by lucas-bruce** [hove-io/navitia#3148]
**Describe the bug**
When going from an adress in Brest (France) to an address in Nantes (France), the best journey uses walking sections accounting for a travel of hundreds of kilometers in a couple minutes.

**To Reproduce**
https://api.navitia.io/v1/journeys?from=-4.4925956%3B48.3857984&to=-1.5339877%3B47.2066542&

**Expected behavior**
The outputed journey should not use quasi teleportation.

**Screenshots**
One screenshot for the full journey, one for the problematic section
![navitia_full_journey](https://user-images.githubusercontent.com/61513403/75368774-8666b880-58c2-11ea-9b93-6d300404b797.jpeg)
![navitia_one_section](https://user-images.githubusercontent.com/61513403/75368776-8797e580-58c2-11ea-92b3-91a16bcf37a5.jpeg)

**Comment by pbougue** [comment]
Hello,

This looks like a data issue, on transfers being a bit too short :frowning_face: 

To make it more easy to reproduce, I fixed datetime (that defaults to "now" otherwise) and coverage (that can depend on what coverage is accessible to a token):
https://api.navitia.io/v1/coverage/fr-nw/journeys?from=-4.4925956%3B48.3857984&to=-1.5339877%3B47.2066542&datetime=20200228T113000

We poked the data team on the subject, as they are watching more the googlegroup.
Letting them reply here (and move the subject on googlegroup if they think it's worthy).

Thanks for the report!
